README.md

```diff
@@ -136,7 +135,7 @@ Available for browsing on https://staging.bitshares.org/
 ### Master (stable)
 When all issues to the current RC, `staging` branch is released to the stable `master` branch.
 
-Available for browsing on https://wallet.bitshares.org/, which is the official reference wallet for Bitshares.
+Available for browsing on https://wallet.bitshares.eu/, which is the official reference wallet for Bitshares.
```

Comment:
Is this a mistake?

Reply:
Hmm, I think I've been having the unfortunate accident of trying to think here. I believed the .eu domain was the one hosting the official wallet, while .org was the ones hosting developer and staging wallets. 

This isn't the case, and it should be .org for wallet.bitshares.org as well.